_ingest-pipelines/accessing-data.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: default
+title: Accessing data in pipeline
+redirect_from:
```

Comment:
Redirects are only necessary for existing pages that have moved. If it's a brand new page (and is not an index page), redirect is not necessary. 
```suggestion
```